frontend/packages/client/src/components/Community/StrategySelectorForm.js

```diff
@@ -129,15 +129,16 @@ export default function StrategySelectorForm({
         </div>
       </div>
       {/* filter elements that will be deleted */}
-      {strategies.map((st, index) => (
-        <StrategySelectorInput
-          index={index}
-          key={`strategy-${index}`}
-          commuVotStra={getStrategyName(allVotingStrategies, st)}
-          onDeleteStrategy={onDeleteStrategy}
-          enableDelete={enableDelete}
-        />
-      ))}
+      {!loadingAllStrategies &&
+        strategies.map((st, index) => (
+          <StrategySelectorInput
+            index={index}
+            key={`strategy-${index}`}
+            commuVotStra={getStrategyName(allVotingStrategies, st)}
+            onDeleteStrategy={onDeleteStrategy}
+            enableDelete={enableDelete}
+          />
+        ))}
```

Comment:
when `allVotingStrategies` is loading it is undefined and `getStrategyName` depends on its value. Adding loadingAllStrategies to check if it's loading to avoid calling the function with `allVotingStrategies === undefined`